### ISSUE TITLE
Ability to specify Tags across all resources

### ISSUE THREAD

**Issue by AlexChesters** [awslabs/aws-cdk#1516]
We have a common set of tags that we apply to all of our AWS resources where possible. These tags are normally used for organisation purposes, such as cost allocation.

Is it possible to define a set of tags that could be applied to all resources that support them? Ideally we'd like an API that looked something like this
```javascript
export default class Stack extends cdk.Stack {
  constructor (parent: cdk.App, name: string, props?: cdk.StackProps) {
    super(parent, name, props)
    this.setGlobalTags({ ComponentName: 'MyComponent', ChargeCode: 'S-XXX' })
  }
}
```

I've read through an issue (https://github.com/awslabs/aws-cdk/issues/91) but it's unclear to me if:
1) Something exists for our use case
2) If it does exist, how it can be used

Thanks in advance 😄

**Comment by AlexChesters** [comment]
Ah I see, thanks @rix0rrr! I did try and see if there was any other relevant issues open but I must have missed that one 😄

**Comment by rix0rrr** [comment]
No problem, just trying to tie things together for easy reference in the future :).

**Comment by eladb** [comment]
Resolved by #1451
